packages/node_modules/@ciscospark/widget-recents/src/components/PopoverContent.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ShallowRenderer from 'react-test-renderer/shallow';
+
+import PopoverContent from './PopoverContent';
+
+const renderer = new ShallowRenderer();
+
+let props;
+let component;
+
+describe('PopoverContent component', () => {
+  beforeEach(() => {
+    props = {
+      currentUser: {
+        id: 'my-id',
+        displayName: 'Snapshot User',
+        img: 'img'
+      },
+      enableAddButton: true,
```

Comment:
Popover content doesn't have any of these props. `onSignoutClick` is missing too.